pkg/kubectl/resource/builder.go

```diff
@@ -607,14 +619,21 @@ func (b *Builder) visitByResource() *Result {
 		isSingleItemImplied = len(b.resourceTuples) == 1
 	}
 
+	result := &Result{
+		singleItemImplied:  isSingleItemImplied,
+		targetsSingleItems: true,
```

Comment:
This never mutates.  Again, I find the value questionable in a method this size.